vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -100,31 +100,113 @@ You have now generated a Spring Boot microservice with database integration and
    * ```/src/main/docker``` contains Docker files to help with local development environments.
 
 ## Configure the Components
-1. From the root directory of the project, edit the `entando.json` and update `microservices/conference-ms` to set the `healthCheckPath` and `dbms`:
+1. From the root directory of the project, edit the `entando.json` file and update `microservices/conference-ms` to set the `healthCheckPath` and `dbms`:
 ```json
    "healthCheckPath":"/management/health",
    "dbms":"postgresql"
 ```
 
-2. Move the generated `conference-table` MFE into the microfrontends directory in the bundle project. If you chose a different entity name, you'll need to adjust these commands accordingly.
+2. Move the generated `conference-table` MFE into the `microfrontends` directory in the bundle project. If you chose a different entity name, you'll need to adjust these commands accordingly.
 ```shell
 ent bundle mfe add conference-table
-mv microservices/conference-ms/ui/widgets/conference/tableWidget microfrontends/ && mv microfrontends/tableWidget microfrontends/conference-table
+mv microservices/conference-ms/ui/widgets/conference/tableWidget/{.,}* microfrontends/conference-table
 ```
+>Note: On some Zsh interfaces, you may receive an invalid argument response that can be ignored as the long as the folders are relocated correctly.
```

Comment:
not really happy with this, unsure how to put it

Reply:
no comment per pending change to nathan's wording